src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -34,13 +34,16 @@
     </EmbeddedResource>
     <EmbeddedResource Include="runtimes.json" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="..\..\.editorconfig" Link=".editorconfig" />
+  </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.7.0" />
     <PackageReference Include="Grpc.Core" Version="1.20.1" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Features" Version="3.1.0" />
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.11020001-fabe022e" />
-    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.5.2-SNAPSHOT" />
+    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.5.4" />
```

Comment:
See @yojagad's PR https://github.com/Azure/azure-functions-java-worker/pull/360. Once that's in, I'll pull that into this PR. For now, I have a local nuget package I've been testing with